packages/components/addon/components/hds/side-nav/list/link.hbs

```diff
@@ -0,0 +1,38 @@
+<Hds::SideNav::List::Item>
+  <Hds::Interactive
+    class="hds-side-nav__list-item-link"
+    @current-when={{@current-when}}
+    @models={{hds-link-to-models @model @models}}
+    @query={{hds-link-to-query @query}}
+    @replace={{@replace}}
+    @route={{@route}}
+    @isRouteExternal={{@isRouteExternal}}
+    @href={{@href}}
+    @isHrefExternal={{@isHrefExternal}}
+    ...attributes
+  >
+    {{#if @icon}}
+      <FlightIcon class="hds-side-nav__list-item-icon" @name={{@icon}} />
+    {{/if}}
+
+    <span class="hds-side-nav__list-item-label hds-typography-body-200 hds-font-weight-medium">
+      {{@label}}
```

Comment:
@KristinLBradley something I forgot in my review: @meirish suggested to use `@text` instead of `@label` to be more consistent with other HDS components. Makes sense to me. What do you think?

Reply:
That's fine. I'll update it.